examples/guestbook/guestbook_ns/redis-slave-service.json

```diff
@@ -0,0 +1,19 @@
+{
+    "kind": "Service",
+    "apiVersion": "v1beta3",
+    "metadata": {
+        "name": "redisslave",
+        "labels": {
+            "name": "redisslave"
+        }
+    },
+    "spec": {
+        "port": 6379,
+        "containerPort": 6379,
+        "protocol": "TCP",
+        "selector": {
+            "name": "redisslave"
+        },
+        "sessionAffinity": "None"
```

Comment:
I see that a number of defaults are specified, presumably as examples. I can understand the reason for that, but it makes our examples look unnecessarily complex, so I think we need to put fully populated objects elsewhere.